common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -178,7 +179,7 @@ export default function SendAssetsForm({ txConfig, onComplete }: IStepComponentP
         .test(
           'check-eth-address',
           translateRaw('TO_FIELD_ERROR'),
-          value => isValidETHAddress(value) || UnstoppableResolution.isValidDomain(value)
+          value => isValidETHAddress(value) || isValidENSName(value)
```

Comment:
Do we really cover the same validation surface as UnstoppableResolution?

Reply:
You're right, we should probably combine the efforts.